gcp-jobs/update-colin-filings/src/update_colin_filings/config.py

```diff
@@ -0,0 +1,76 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""All of the configuration for the service is captured here."""
+
+import os
+import sys
+
+from dotenv import find_dotenv, load_dotenv
+
+load_dotenv(find_dotenv())
+
+
+class _Config:
+    """Base class configuration."""
+
+    PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
+    
+    JOB_TOTAL_LIMIT = int(os.getenv("JOB_TOTAL_LIMIT", "500"))
+    JOB_BATCH_LIMIT = int(os.getenv("JOB_BATCH_LIMIT", "50"))
```

Comment:
This and the one above should be in the .env.sample

Reply:
Updated